.circleci/config.yml

```diff
@@ -12,7 +12,7 @@ jobs:
         - develop
         - /feature-.*/
         - JP
-        - /*_Old/
+        - /.*_Old/
```

Comment:
Well, Buildbot fixes have to go into JP branch and merged to all the other branches, SOOOO

Reply:
:eyes:

Reply:
Already done in fb4fda0501e89c47ce0958d9e3c1c64cda599a16

Reply:
shouldn't cause issues on merge right?

Reply:
No, but your name will be on the blame list, and do you really want to?

Reply:
true.... i'm unsure how to snip it out of the PR, ur gonna have to teach me

Reply:
You can use `git rebase` to rewrite commits
But you need to use `git push --force` to make GitHub take your history changes